src/main/java/org/gridsuite/modification/server/modifications/ModificationUtils.java

```diff
@@ -814,5 +814,137 @@ public void checkMaxReactivePowerGreaterThanMinReactivePower(MinMaxReactiveLimit
             throw new NetworkModificationException(exceptionType, errorMessage + "maximum reactive power " + maxReactivePower + " is expected to be greater than or equal to minimum reactive power " + minReactivePower);
         }
     }
+
+    private static NetworkModificationException makeEquipmentException(NetworkModificationException.Type errorType,
```

Comment:
remove static keyword and use singleton instance, to be consistent with the other method calls
same remark for the following methods ...